specification/_json_spec/ingest.delete_ip_location_database.json

```diff
@@ -0,0 +1,28 @@
+{
+  "ingest.delete_ip_location_database": {
```

Comment:
@joegallo I'm slightly confused by those new APIs. Based on https://github.com/elastic/elasticsearch/pull/114548/files#diff-e691792ba34345b9c60eac06a6730438c8265376839f01e598b4f02e39f8a1ac, my understanding is that right now they are strictly equivalent to the GeoIP APIs. Will they do something different at some point? When should we expose them to clients and serverless docs?

Reply:
In the eventual future, the `geoip` APIs will be deprecated and removed (there is no plan to do this at the moment, but given the fullness of time it will happen 'one day'), but the `ip_location` APIs will stay around forever. In the near and intermediate term, it's the same behavior available under two names.